src/mumble/AudioOutput.h

```diff
@@ -150,6 +150,10 @@ class AudioOutput : public QThread {
 	/// @param modifiedAudio Pointer to bool if audio has been modified or not and should be played
 	void audioOutputAboutToPlay(float *outputPCM, unsigned int sampleCount, unsigned int channelCount,
 								unsigned int sampleRate, bool *modifiedAudio);
+
+	void bufferInvalidated(AudioOutputBuffer *);
+
+	void bufferPositionChanged(AudioOutputBuffer *, float x, float y, float z);
```

Comment:
Shouldn't these be private signals?

Reply:
According to the pre-processor private singals do not exist :P

Reply:
Ah well :sweat_smile: :shrug: